utils/efushell/efustatus.py

```diff
@@ -16,9 +16,9 @@
 metrics = Metrics(args.i, args.p)
 
 print(metrics._get_efu_command("RUNTIMESTATS"))
-
 print(metrics._get_efu_command("VERSION_GET"))
 print(metrics._get_efu_command("DETECTOR_INFO_GET"))
+print(metrics._get_efu_command("CALIB_MODE_GET"))
```

Comment:
Shouldn't the CALIB_MODE_SET also added here?

Reply:
No, this is intentional, I am only interested in getting the status in this command